chart/config.go

```diff
@@ -2,11 +2,10 @@ package chart
 
 import (
 	"github.com/giantswarm/helmclient"
-
-	"github.com/giantswarm/e2esetup/k8s"
```

Comment:
Dropped duplicate k8s setup package since it's present in k8sclient; it was misleading many to unnecessarily depend on e2esetup for just setup purpose which was creating cyclic dependencies.